models/articles.model.js

```diff
@@ -11,4 +11,20 @@ exports.getArticleWithId = (articleId) => {
             if (!results.rowCount) return Promise.reject({ status: 404, msg: 'Article not found'})
             return results.rows[0]
         })
+}
+
+exports.getArticles = () => {
+    return db.query(`
+        SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(*) as comment_count
+        FROM articles
+        LEFT JOIN comments ON articles.article_id = comments.article_id
+        GROUP BY articles.article_id
+        ORDER BY articles.created_at DESC
+        `)
+        .then(({ rows: articles }) => {
```

Comment:
you dont need this here. you can adjust your sql query to " COUNT(*)::INT AS comment_count"